lib/models/ecs-model-deployer.ts

```diff
@@ -29,6 +29,7 @@ export type ECSModelDeployerProps = {
 } & BaseProps;
 
 export class ECSModelDeployer extends Construct {
+    readonly fn: Function;
```

Comment:
let's make this name more descriptive so that we can identify what it is, something like "ecsModelDeployerFunction" or similar, that way if we need to grab another item out of this construct in the future, we have less have a chance of ambiguously named resources. (and we'd need to change any of the references that use it too)